src/components/tabs/tabs.js

```diff
@@ -218,10 +233,25 @@ Tabs.prototype.activateNextTab = function () {
 }
 
 Tabs.prototype.activatePreviousTab = function () {
+  // IE doesn't support 'previousElementSibling' this code polyfills IE8
+  // https://developer.mozilla.org/en-US/docs/Web/API/NonDocumentTypeChildNode/previousElementSibling#Polyfill_for_Internet_Explorer_8
+  // Source: https://github.com/Alhadis/Snippets/blob/master/js/polyfills/IE8-child-elements.js
+  if (!('previousElementSibling' in document.documentElement)) {
+    Object.defineProperty(Element.prototype, 'previousElementSibling', {
+      get: function () {
+        var e = this.previousSibling
+        while (e && e.nodeType !== 1) {
+          e = e.previousSibling
+        }
+        return e
+      }
+    })
+  }
+
   var currentTab = this.getCurrentTab()
   var previousTabListItem = currentTab.parentNode.previousElementSibling
```

Comment:
out of curiosity, if we changed `previousElementSibling` to `previousSibling` would this resolve the issue?

Reply:
Nice one, I can remove the polyfill 😄

Reply:
Should test this change carefully – I wasn't previously aware of the difference but it looks like next/previousSibling [may return the whitespace between elements](https://stackoverflow.com/questions/24226571/what-is-the-difference-between-node-nextsibling-and-childnode-nextelementsibling)

Reply:
Yes I did run into so problems while testing and its also one of the reason the automated tests were failing too. I have since reintroduced the polyfill.